src/config.js

```diff
@@ -2,9 +2,26 @@ const webpackConfig = require('../webpack');
 const defaultConfig = require('./vussr.config.default');
 
 class Config {
+
   constructor(config, cliOptions) {
     const nockOptions = this.getNockOptions(cliOptions);
     this.config = Object.assign({}, defaultConfig, config, nockOptions);
+    this.config.isCDN = this.isUrl(this.config.assetsUrlPath);
+    if(!this.config.isCDN) {
+      this.config.assetsUrlPath = this.cleanRelativePath(this.config.assetsUrlPath);
+    }
+  }
+
+  cleanRelativePath(val) {
+    return val.replace(/^\/?/, '/').replace(/\/?$/, '/');
+  }
+
+  isUrl(string) {
```

Comment:
It doesn't use `this` at all, can we just make it a function instead of a method?

Reply:
It's yours. I just inserted you the new config variable. ;-)

Reply:
@mkai I would keep it as a method of `Config` for code cleanliness, i.e. keep everything inside one place (the class). I would not mix classical programming with functional programming, at least not in one file.